### PR TITLE
Single language request merge into develop

### PR DESCRIPTION
**Pull request by beattie19**
I figured it would be good to get this up so Gergo can work on CSS if he is happy to, just basing that off of trello labels. 

couple of notes. - needs refactoring but not too confident on using 'this' and to move some of that stuff out of the render function, proved a little too difficult for me but it at least prints all information to the screen. I added a button to the page only because I wanted to test the render but wasn't too sure how to do it on the ruby button click right now. I can look at that later or tomorrow.

### CODE REVIEW

File: client/src/views/indiv_lang_view.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+var AjaxRequest = require('../services/ajax_request');
+var IndivLangView = function() {
+
+}
+
+IndivLangView.prototype.render = function(language) {
+    var section = document.getElementById('main-view');
```
Comment (CJHunter91) [inline client/src/views/indiv_lang_view.js]: You should probably empty any html in the main view at this point. I created the history view which will populate main view on window load. That way when the button is clicked it doesn't add it to the end of the history text or any other elements that might have been appended. @beattie19

**Comment by CJHunter91** [comment]
Yeah that looks ok at the moment. We can think about how to more dynamically create the request url's later. Other than that bit of code I reviewed it will be ready to merge.

**Comment by beattie19** [comment]
Nice one. Do you want to merge the pull request as we mentioned we wouldn't merge our own requests

**Review by CJHunter91 — CHANGES_REQUESTED** [review]
One bit of code may need added.

**Review by CJHunter91 — APPROVED** [review]
That looks good.

**Pull request merged**